plugins/techdocs-backend/src/DocsBuilder/builder.ts

```diff
@@ -136,13 +137,18 @@ export class DocsBuilder {
      */
 
     this.logger.info(
-      `Step 2 of 3: Generating docs for entity ${serializeEntityRef(
+      `Step 2 of 3: Generating docs for entity ${stringifyEntityRef(
         this.entity,
       )}`,
     );
 
     // Create a temporary directory to store the generated files in.
-    const tmpdirPath = os.tmpdir();
+    const config = await loadBackendConfig({
+      argv: process.argv,
+      logger: getRootLogger(),
+    });
```

Comment:
The plugin already has a `config` instance provided by the Backstage backend to techdocs backend (inside `createRouter` in `plugins/techdocs-backend/src/service/router.ts`).

I think when an instance of `DocsBuilder` is being created inside the `createRouter`, it should forward the `config` instance as well, which can be used here. :)

Reply:
Yep! The isn't exported either so should be np passing in the config instance in `router.ts` without it being a breaking change.

Reply:
Yep so adding it effectively to `DocsBuilderArguments`